app/controllers/projects/samples_controller.rb

```diff
@@ -50,7 +58,7 @@ def update
       respond_to do |format|
         if ::Samples::UpdateService.new(@sample, current_user, sample_params).execute
           flash[:success] = t('.success')
-          format.html { redirect_to namespace_project_sample_path(id: @sample.id) }
```

Comment:
You can also revert this change and add a data: {turbo: false} to the form signature in `samples/edit.html.erb`. It will look like this:

`<%= form_with(model: @sample, url: namespace_project_sample_path(id: @sample.id), data: {turbo: false}, method: :patch) do |form| %>`

This will properly redirect from the edit sample page to the show sample page

Reply:
Fixed in 0fb8fc1c66eb29d81ced91306361fadc7cb1d8b3